src/modules/olcs/olcsmodule.js

```diff
@@ -4,6 +4,10 @@
 goog.module('ngeo.olcs.olcsModule');
 goog.module.declareLegacyNamespace();
 
+goog.require('ngeo.Debounce');
+goog.require('ngeo.Location');
+goog.require('ngeo.StateManager');
```

Comment:
Why require the dependencies here? You should require them in the service that use them.